packages/core-components/src/layout/Page/Page.tsx

```diff
@@ -52,7 +52,7 @@ export function Page(props: Props) {
         page: baseTheme.getPageTheme({ themeId }),
       })}
     >
-      <div className={classes.root}>{children}</div>
+      <main className={classes.root}>{children}</main>
```

Comment:
I wonder if it's better to make this change in `packages/core-components/src/layout/Sidebar/Page.tsx` instead? It's not really guaranteed that the `Page` component is used everywhere

Reply:
Can you provide an example of where the `Page` component isn't used? The issue with moving it there is that it would also wrap the `<nav>`, which defeats some of the benefits of having it.

Reply:
Ah alright, let's add it here then and if the `Page` component is not used then it'd be up to the replacement to make sure `<main>` is used.